kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -1321,6 +1321,15 @@ messages:
       {
          iSay_type = Nth(client_msg,2);
          sStr_said = Nth(client_msg,3);
+
+         % don't send oversized chat strings
+         if (StringLength(sStr_said > MAX_CHAT_LEN))
```

Comment:
if (StringLength(sStr_said > MAX_CHAT_LEN))
should be

    if (StringLength(sStr_said) > MAX_CHAT_LEN)
Applies to the one below also